flake8_import_order/flake8_linter.py

```diff
@@ -46,11 +48,19 @@ def add_options(cls, parser):
             default=DEFAULT_IMPORT_ORDER_STYLE,
             action="store",
             type="string",
-            help=("Style to follow. Available: "
-                  "cryptography, google, smarkets, appnexus, pep8"),
+            help=("Style to follow. Available: " +
+                  ", ".join(cls.list_available_styles())),
             parse_from_config=True,
         )
 
+    @staticmethod
+    def list_available_styles():
+        entry_points = pkg_resources.iter_entry_points(
+            'flake8_import_order.styles'
+        )
+        for entry_point in entry_points:
+            yield entry_point.name
+
     @classmethod
     def parse_options(cls, options):
```

Comment:
You should be finding the entry-point here and passing it through on the `options` object.